pkg/control/control.go

```diff
@@ -378,61 +383,68 @@ LOOP:
 				break LOOP
 			default:
 			}
-
-			log.Printf("begin to run %s nemesis generator", g.Name())
-			ops := g.Generate(c.cfg.Nodes)
-
-			wg.Add(len(ops))
+			var (
+				ops = g.Generate(c.cfg.Nodes)
+				g   errgroup.Group
```

Comment:
It's better to use another variable name instead of overwriting `g`

Reply:
Good catch, I didn't notice that.

Reply:
Updated.